remail/email_api/service.py

```
@@ -0,0 +1,490 @@
+                                html_files = html_parts)]
+        return listofMails
+
+    def get_deleted_emails(self,uids:list[str])->list[str]:
+        if not self.logged_in: 
+            return None
+        listofUIPsIMAP = []
+        for mailbox in [folder[2] for folder in self.IMAP.list_folders()]:
+            print(mailbox)
+            self.IMAP.select_folder(mailbox)
+            messages_ids = self.IMAP.search(["ALL"])
+            for msgid,message_data in self.IMAP.fetch(messages_ids,["RFC822"]).items():
```

Ruff (0.8.0)

Loop control variable `msgid` not used within loop body

Rename unused `msgid` to `_msgid`

(B007)